src/llmq/chainlocks.cpp

```diff
@@ -665,8 +666,6 @@ void CChainLocksHandler::Cleanup()
             ++it;
         }
     }
-
-    lastCleanupTime = GetTimeMillis();
```

Comment:
can it change behavior in case if `Cleanup()` will take significant time?

It can cause of new iteration of `Cleanup` start immediately after previous iteration is done if `cs_main` will be busy for awhile and cleaning process will start with a delay.

Reply:
possible if you debug application and run it step-by-step, but I guess that is not really important